src/agent/multitype_queue/include/imultitype_queue.hpp

```diff
@@ -133,4 +135,12 @@ class IMultiTypeQueue
      * @return int The number of items in the queue.
      */
     virtual int storedItems(MessageType type, const std::string moduleName = "") = 0;
+
+    /**
+     * @brief Returns the number of bytes stored in the queue.
+     *
+     * @param type The type of the queue.
+     * @return size_t The number of bytes in the queue.
+     */
+    virtual size_t sizePerType(MessageType type) = 0;
```

Comment:
No `moduleName` here?

Reply:
In the meantime the fuction that make use of this method doesn't use `moduleName`. That's why at first we assume to measure only the size occupied by a single DB. 
IMO It can be enhanced in the future.